test/blackbox-tests/test-cases/oxcaml/library_parameter.t

```diff
@@ -61,9 +61,35 @@ We rebuild it and ensure we have the new parameter available.
   $ ocamlobjinfo _build/default/param/.param_intf2.objs/byte/param_intf2.cmi | grep "Is parameter"
   Is parameter: YES
 
+We create a third parameter with multiple modules. The test should fail as we
+limit the parameter to one module now.
+
+  $ echo "(package (name multiple_param))" >> "dune-project"
+  $ mkdir multiple
+  $ cat > "multiple/dune" << EOF
+  > (library_parameter
+  >  (public_name multiple_param)
+  >  (name multiple_param)
+  >  (modules multiple_param data))
+  > EOF
+  $ cat > "multiple/data.mli" << EOF
+  > type t = int
+  > EOF
+  $ cat > "multiple/multiple_param.mli" << EOF
+  > type t = string
+  > EOF
+
+  $ dune build
```

Comment:
I'm a bit confused, it doesn't seem like this failed?

Reply:
Yep, and that's a problem I need to fix for this PR.

Reply:
I have pushed the fix: https://github.com/ocaml/dune/pull/11963/files#diff-97a3106adc9720a309352f46b8d89a0bdf768edd1ebf5709d49f925fce3aa07fR438